server/src/main/java/com/lunchtime/entity/Event.java

```diff
@@ -0,0 +1,35 @@
+package com.lunchtime.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.net.URI;
+import java.sql.Date;
+
+@ToString
+@Setter
+@Getter
+@Entity
+@Table(name = "events")
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private long restaurantId;
+    private Date date;
+    private URI eventImage;
+    private String eventName;
+    private String category;
+    private String description;
+
+    /**
+     * An empty constructor is needed to create a new instance via
+     * reflection by persistence framework.
+     */
+    public Event() {
+    }
```

Comment:
I did similar class, and code works without empty constructor. Is it really need? As Java class has empty constructor by default.